File: Java Programs/Chapter 1/Factorial_Iterative/src/factorial_iterative/Factorial_Iterative.java

package factorial_iterative;

//Program to find the factorial of a number with value within the range of a long integer using iterative method.
import java.util.*;

public class Factorial_Iterative
{
	public static void findFactorial(int n)
	{
		long factorial=1;
		for(int i=2;i<=n;i++)
		{
			factorial=factorial*i;
		}
		System.out.print("Factorial of "+n+": "+factorial);
	}
	public static void main(String args[])
	{
		Scanner oin=new Scanner(System.in);
		System.out.print("Enter the number for finding factorial: ");
		int n=Integer.parseInt(oin.nextLine());
		findFactorial(n);
	}
}

File: Python Programs/unit1/FactorialIterative.py
from __future__ import print_function


def factorial(n):
    factorial = 1
    for i in range(2, n + 1):
        factorial = factorial * i
    return factorial


print("Enter the number to find factorial")
n = int(raw_input())
print("FACTORIAL : %d" % factorial(n))
